app/views/corrections/CheckVatPayableAmountView.scala.html

```diff
@@ -14,12 +14,15 @@
  * limitations under the License.
  *@
 
+@import components.ButtonGroup
+
 @this(
         layout: templates.Layout,
         govukSummaryList: GovukSummaryList,
         govukButton: GovukButton,
```

Comment:
Is this needed anymore?

Reply:
I added it as was getting a compile error:

[error] /Users/leepowell/repo/hmrc/one-stop-shop-returns-frontend/app/views/corrections/CheckVatPayableAmountView.scala.html:23:17: not found: type ButtonGroup
[error]         button: ButtonGroup
[error]                 ^